pmd/pm_vty.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * VTY library for Path Monitoring Daemon
+ * Copyright (C) 6WIND 2019
+ *
+ * This file is part of FRR.
+ *
+ * FRR is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * FRR is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+#ifndef __PM_VTY_H__
+#define __PM_VTY_H__
+
+extern void pm_vty_init(void);
```

Comment:
Could you move this function declaration to another header? I think it is a bit wasteful to have an entire header for a single function.